### PR TITLE
[MOV] slides: Rename and split files according to their underlying models

### PR DESCRIPTION
**Pull request by qmo-odoo**
Purpose : Cleaning the module to have an easier time turning it into courses

Task: #1909281

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_slides/__manifest__.py [hunk 1]
```diff
@@ -19,11 +19,17 @@
...
     'data': [
+        'views/assets.xml',
         'views/res_config_settings_views.xml',
-        'views/website_slides.xml',
-        'views/website_slides_embed.xml',
-        'views/website_slides_backend.xml',
-        'data/website_slides_data.xml',
+        'views/website_slides_templates.xml',
+        'views/website_slides_embed_templates.xml',
+        'views/website_slides_slide_views.xml',
```
Comment (tde-banana-odoo) [inline addons/website_slides/__manifest__.py]: Even if module name is website slides, model are named slide.channel and slide.slide. View files should therefore be slide_channel_views.xml, slide_slides_views.xml, ... Same for data or other files (mail_data.xml, website_data.xml, ...).

Website_slide_* prefix should be used whenever no main model applies to the files content. For example, a generic templates files for frontend: website_slides_templates.xml .

**Comment by tde-banana-odoo** [comment]
@robodoo r+

File: addons/website_slides/static/src/js/slides_embed.js [hunk 2]
```diff
@@ -1,5 +1,50 @@
 /*global $, _, PDFSlidesViewer*/
+odoo.define('website_slides.slides_embed', function (require) {
```
Comment (qsm-odoo) [inline addons/website_slides/static/src/js/slides_embed.js]: Are you sure about moving this in this file without any adaptation ? The file is not in the same asset bundle...

File: addons/website_slides/static/src/js/slides_embed.js [hunk 3]
```diff
@@ -1,5 +1,50 @@
...
+    "use strict";
+    
+    var time = require('web.time');
+    var Widget = require('web.Widget');
+    require('website_slides.slides');
+    
+    (function () {
```
Comment (qsm-odoo) [inline addons/website_slides/static/src/js/slides_embed.js]: Why wrapping all your code in a function call ?

File: addons/website_slides/static/src/js/slides_embed.js [hunk 4]
```diff
@@ -1,5 +1,50 @@
...
+            var $iframe = $(this);
+            var max_page = $iframe.contents().find('#page_count').val();
+            slides.page_widgets['social_embed'] = new SlideSocialEmbed($iframe, max_page).setElement($('.oe_slide_js_embed_code_widget'));
+        });
+    
+    })();
+    
+});
 /**
 This file is a minimal version of the PDFViewer widget.
```
Comment (qsm-odoo) [inline addons/website_slides/static/src/js/slides_embed.js]: ...

File: addons/website_slides/views/assets.xml [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" ?>
+<odoo>
+    <data>
+
+        <template id="assets_frontend" inherit_id="website.assets_frontend" name="Slide">
+            <xpath expr="." position="inside">
+                <link rel="stylesheet" type="text/scss" href="/website_slides/static/src/scss/website_slides.scss" t-ignore="true"/>
+                <!-- Js widgets, ... order loading files is IMPORTANT -->
```
Comment (qsm-odoo) [inline addons/website_slides/views/assets.xml]: Why would that be important ?

**Pull request closed without merging**